### DISCUSSION TITLE
wiki/G__GraphicalEditorTutorial

### DISCUSSION THREAD

**Discussion by giscus** [gama-platform/gama-platform.github.io#206] [General]
# wiki/G__GraphicalEditorTutorial

This tutorial illustrates how to create simple agents and make them move in their environment.

https://gama-platform.org/wiki/G__GraphicalEditorTutorial

**Comment by mbalagueraj** [comment]
When trying to create the "people" layer for "my_display", if I click the button "add", a layer with name "layer" is added to the list, however the editor doesn´t appear and nothing happens when trying to use the button "edit" in order to set the layer for "people" species. Thank for your help.

**Reply by mbalagueraj** [reply]
I have followed step by step this tutorial and implement the project side by side in my installed 1.9.2 GAMA, however in the step corresponding to the 6th GIF, after clicking on the button to create a layer, nothing happens and I am not able to specify the "people" species for the created layer. In advance, thanks for any advice.

**Reply by lesquoyb** [reply]
maybe @ptaillandier can have a look ?

**Comment by NgomsemNgueramadji** [comment]
How to integrate diagram tool  model under GAMA?
